links/src/iosMain/kotlin/com/splendo/kaluga/links/manager/PlatformLinksHandler.kt

```diff
@@ -29,10 +31,10 @@ actual class PlatformLinksHandler : LinksHandler {
         return NSURLConnection.canHandleRequest(NSURLRequest.requestWithURL(_url))
     }
 
-    override fun extractQueryAsList(url: String): List<NameValue> {
+    override fun extractQueryAsList(url: String): ParametersNameValue {
         val urlComponents = NSURLComponents(url)
-        val queryItems = urlComponents.queryItems as List<NSURLQueryItem>? ?: return emptyList()
-
-        return queryItems.map { it.name to (it.value as Any) }
+        val queryItems = urlComponents.queryItems as List<NSURLQueryItem>? ?: return emptyMap()
```

Comment:
As list or empty map? I think it's fine to keep emptyList

Reply:
its return statement if it cannot be cast as `List<NSURLQueryItem>`

Reply:
Yes, but here is inconsistency, you try to case to list but return map as a fallback. If you can't cast you have to create same empty list, or I miss something?

Reply:
89683b0